Content.Server/Cloning/CloningSystem.cs

```diff
@@ -113,6 +101,26 @@ public bool TryCloning(EntityUid original, MapCoordinates? coords, ProtoId<Cloni
         return true;
     }
 
+    public bool TryCloneComponents(EntityUid original, EntityUid clone, HashSet<String> componentsToCopy)
```

Comment:
```suggestion
    public void CloneComponents(EntityUid original, EntityUid clone, HashSet<String> componentsToCopy)
```
no way for this to return false
I would also make it use a cloning settings prototype instead of a HashSet